package.json

```diff
@@ -6,6 +6,10 @@
   ],
   "scripts": {
     "build": "lerna run build",
+    "build:visual": "docker build -t screenshot:latest ./;",
+    "test:visual": "yarn build:visual && docker run -t screenshot:latest yarn test:integration",
+    "copy:test:visual": "echo 'do something to get __diff_output__ out of the container'",
```

Comment:
Not sure if you pushed latest changes, but I don't see `__diff_output__` folder locally after diffs were written in Docker.

Reply:
Fixed by changing the `docker run` calls to mount the local directory directly in the container to keep them in sync